docs/how-to/crafting/configure-package-information.rst

```diff
@@ -41,6 +41,11 @@ The placeholder values looks similar to this:
       my-part:
         plugin: nil
 
+    apps:
+      my-app:
+        command: usr/bin/my-app
+        common-id: org.test.myapp
+
```

Comment:
```suggestion
```

This code block represents the output of `snapcraft init`, which doesn't currently produce these.